wordpress-mobile/WordPress-iOS
# UGC: Hide Blocked Sites's posts in Reader Search

Part of #19981

## Related PR
This PR is a continuation of the following PR:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/20023

## Description

Prior to this PR, it was possible to see a site's posts in Site Posts List screen even if the site is blocked. This PR fixes this issue. 

| Before | After |
| ------ | ------ | 
| ![](https://user-images.githubusercontent.com/9609223/218124354-32ac1fec-76ef-4876-b85a-70f837ab7068.png) | ![](https://user-images.githubusercontent.com/9609223/218124537-f8c610db-c93c-43e7-93bd-66decb4575cd.png) |

## Test Instructions

1. Block a site following the Test Instructions steps in this [PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/20023).
2. Head to Reader > Search 🔍  screen.
3. Search for the blocked site.
4. Head to "Sites" tab.
5. **Expect** to see the blocked site.
6. Tap on the site.
7. **Expect** to see "No posts" empty state message.

## Regression Notes
1. Potential unintended areas of impact
Search for un-blocked sites and make sure you can see their posts.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Existing Unit Tests and Manual Tests.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
